stake-pool/py/stake_pool/constants.py

```diff
@@ -17,6 +17,9 @@
 MINIMUM_ACTIVE_STAKE: int = MINIMUM_DELEGATION
 """Minimum active delegated staked required in a stake account"""
 
+METADATA_PROGRAM_ID: PublicKey = PublicKey("metaqbxxUerdq28cj1RbAWkYQm3ybzjb6a8bt518x1s")
+"""Public key that identifies the SPL Token Metadata program."""
```

Comment:
nit: this is the metaplex metadata program
```suggestion
"""Public key that identifies the Metaplex Token Metadata program."""
```

Reply:
Yes sure, C/P mistake 😅